taworks/taform/static/taform/js/app-jquery.js

```diff
@@ -1,4 +1,7 @@
 $(document).ready(function() {
+
```

Comment:
3 extra lines removal and same below.

Reply:
I think red symbolizes an error once it's been made (like how it happens after we submit and they don't fill it out) but @amyleblond @smwatts what do you think?

Reply:
not standard for it to be reed